x-pack/plugins/security_solution/public/entity_analytics/components/asset_criticality/translations.ts

```diff
@@ -19,25 +19,25 @@ export const CRITICALITY_LEVEL_TITLE: Record<CriticalityLevel, string> = {
   normal: i18n.translate(
     'xpack.securitySolution.entityAnalytics.assetCriticality.levelTitle.normal',
     {
-      defaultMessage: 'Normal',
+      defaultMessage: 'Medium impact',
```

Comment:
@tiansivive FYI I'm changing these translations, which will change all instances of `Asset CriticalityBadge` 👍

Reply:
Actually I'm not, Mark did that on #176802.